src/app/pages/fund/fund.layout.tsx

```diff
@@ -35,12 +50,12 @@ export function FundLayout({ address }: FundLayoutProps) {
           maxWidth="544px"
           textAlign={['left', 'center']}
         >
-          Choose an exchange to fund your account with Stacks (STX) or deposit from elsewhere.
-          Exchanges with “Fast checkout” make it easier to purchase STX for direct deposit into your
-          wallet with a credit card.
+          Choose an exchange to fund your account with {name} ({nameAbbr}) or deposit from
+          elsewhere. Exchanges with “Fast checkout” make it easier to purchase {nameAbbr} for direct
+          deposit into your wallet with a credit card.
         </styled.span>
       </Stack>
-      <FiatProvidersList address={address} />
+      <FiatProvidersList address={address} symbol={symbol} />
```

Comment:
I see this was done before your changes, but I notice `<FiatProvidersList/>` is used directly in the layout component. 

Should better be passed in, as it's a smart component that FundLayout is aware of (when it should only handle the layout).

```ts
<FundLayout>
  <FiatProvidersList address={address} symbol={symbol} />
</FundLayout>
```

would be an improvement if you could make that change